frontend/public/style/mixin/_break-word.scss

```diff
@@ -0,0 +1,6 @@
+// Generic mixin for break-word. Note if element is a flex child then min-width: 0% is necessary for it to wrap.
+
+@mixin co-break-word() {
+  overflow-wrap: break-word; // new name as per the CSS3 spec
+  word-wrap: break-word; // IE 11/16/17 requires legacy name "word-wrap" rather than "overflow-wrap".
```

Comment:
What is IE 16/17? Edge?

Reply:
yes

Reply:
https://caniuse.com/#search=overflow-wrap

Reply:
I'd suggest updating the comment to say Edge 16/17

Do we know what "partial support" in that table means? It would be nice to not need the mixin and just use overflow-wrap

Reply:
Should we use a placeholder instead of a mixin?

http://thesassway.com/intermediate/understanding-placeholder-selectors

Reply:
Per the caniuse notes - "Partial support refers to requiring the legacy name "word-wrap" (rather than "overflow-wrap") to work."

I'll look into the placeholder option.

Reply:
@spadgett 
While `@extend` placeholder selectors themselves aren't in the generated css. They aren't able to be used within `@media` blocks.

```
@media screen {
    .foo {
      @extend %extend-co-break-word;
    }
  }
```
generates ... "You may not @extend an outer selector from within @media.
You may only @extend selectors within the same directive."

Also from 
https://tech.bellycard.com/blog/sass-mixins-vs-extends-the-data/ mixins
based on this persons testing for "...smallest file size, fastest load time, and simplest to use"
mixins are usually preferred.

"Extends can break the cascade, cause you to increase specificity as a result, and spread your styles out all over the place..."

Reply:
@spadgett Would you prefer I switch to `@extend` or stay as a mixin?

Reply:
That's too bad. Thanks for checking. I'm fine with the mixin.